cabal-testsuite/PackageTests/NewConfigure/cabal.project.local

```diff
@@ -0,0 +1,26 @@
+verbose: verbose +nowrap +markoutput
```

Comment:
Okay. I ran the test, and whatever the output was, I just committed it. I can keep this simple.